apps/showcase/project.json

```diff
@@ -192,6 +204,54 @@
         "^build-builders"
       ]
     },
+    "generate-dark-theme": {
+      "executor": "@o3r/design:generate-css",
+      "outputs": [
+        "{options.defaultStyleFile}",
+        "{options.metadataOutput}",
+        "{projectRoot}/src/**/*.scss"
+      ],
+      "inputs": [
+        "{projectRoot}/src/style/dark-theme/dark-theme.*.json"
+      ],
+      "options": {
+        "defaultStyleFile": "apps/showcase/src/style/dark-theme/dark-theme.scss",
+        "designTokenFilePatterns": "apps/showcase/src/style/dark-theme/dark-theme.*.json",
+        "rootPath": "apps/showcase/src"
+      },
+      "configurations": {
+        "watch": {
+          "watch": true
+        }
+      },
+      "dependsOn": [
+        "^build-builders"
+      ]
+    },
+    "generate-random-theme": {
+      "executor": "@o3r/design:generate-css",
+      "outputs": [
+        "{options.defaultStyleFile}",
+        "{options.metadataOutput}",
+        "{projectRoot}/src/**/*.scss"
+      ],
+      "inputs": [
+        "{projectRoot}/src/style/dark-theme/random-theme.*.json"
+      ],
+      "options": {
+        "defaultStyleFile": "apps/showcase/src/style/random-theme/random-theme.scss",
+        "designTokenFilePatterns": "apps/showcase/src/style/random-theme/random-theme.*.json",
+        "rootPath": "apps/showcase/src"
+      },
+      "configurations": {
+        "watch": {
```

Comment:
should be added in `serve` multi-watch (you can also add `generate-dark-theme:watch`)